ports/gumbo/CMakeLists.txt

```
@@ -24,11 +24,27 @@ endif (MSVC)
 add_library(gumbo ${gumbo_srcs})
 
 install(
-  TARGETS gumbo
+  TARGETS gumbo  EXPORT unofficial-gumboTargets
   RUNTIME DESTINATION bin
   LIBRARY DESTINATION lib
   ARCHIVE DESTINATION lib
 )
+
+install(EXPORT unofficial-gumboTargets
+    NAMESPACE unofficial::gumbo::
+    DESTINATION share/unofficial-gumbo
+    )
+
+file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/unofficial-gumbo-config.cmake.in"
+[[
+include("${CMAKE_CURRENT_LIST_DIR}/unofficial-gumboTargets.cmake")
+]])
+
+configure_file("${CMAKE_CURRENT_BINARY_DIR}/unofficial-gumbo-config.cmake.in" "${CMAKE_CURRENT_BINARY_DIR}/unofficial-gumbo-config.cmake" @ONLY)
```

```suggestion
install(EXPORT unofficial-gumbo-config
    NAMESPACE unofficial::gumbo::
    DESTINATION share/unofficial-gumbo
    )
```
Since this port doesn't have any extra dependencies, I'd like to export the config file directly.